documentation/doc-Migration_Toolkit_for_Virtualization/master.adoc

```diff
@@ -124,6 +124,19 @@ include::modules/mtv-ui.adoc[leveloffset=+2]
 include::modules/mtv-overview-page.adoc[leveloffset=+2]
 include::modules/mtv-settings.adoc[leveloffset=+2]
 
+[id="ways-to-create-migration-plans-ui"]
+=== Ways to create migration plans and components using the {ocp-short} web console
+
+There are two ways to create migration plans and components using the {ocp-short} web console:
+
+* The _step-by-step_ method, in which you create plans components - such as providers and network mappings - and then combine them into a migration plan. Although you do not have to create the components in a definite order, the steps are listed below in the recommended order of work.
```

Comment:
hmm, in both ways you need to do the step-by-step thingy

different:
a. in the quick create you skip the first step of choosing the source provider and the virtual machines to migrate.
in the quick create you start from step 2 in the wizard.

same:
a. in both ways you need to first create a source provider.
b. in both ways you do not need to create the mappings or hooks, ( not implemented yet - in both ways you can pre-create the mappings and hooks )

Reply:
@yaacov @ahadas @anarnold97  Please help me out here with names for the two approaches.
In the first approach, the user defines everything -- source, target, network mapping, source mapping -- that is needed to create a migration plan. 
In the second approach, the quick plan, all the user has to do is identify the source -- either by  creating or choosing it and MTV supplies editable choices for the rest. 

I thought I would differentiate between the two by calling the first "step-by-step" but Kobi points out the quick plan also has steps. What should I call the first approach? "Detailed plan" maybe? Maybe I don't need a name -- just use what I wrote in this note. What do you think? 

I want to introduce the approaches, go through the detailed one, and then the quick plan, with both scenarios.